scanner/matcher/updater/distribution/updater.go

```diff
@@ -64,27 +70,36 @@ func (u *Updater) Known() []claircore.Distribution {
 // Start begins the update proces.
 func (u *Updater) Start() error {
 	ctx := zlog.ContextWithValues(u.ctx, "component", "matcher/updater/distribution/Updater.Start")
-
-	zlog.Info(ctx).Msg("starting initial update")
-	if err := u.update(ctx); err != nil {
-		zlog.Error(ctx).Err(err).Msg("errors encountered during updater run")
-	}
-	zlog.Info(ctx).Msg("completed initial update")
-
-	t := time.NewTimer(updateInterval)
+	// Schedule the initial update to occur randomly anywhere from now up to the
+	// specified limit minutes, to reduce chances of simultaneous runs.
+	d := time.Duration(rand.Float64() * float64(1*time.Minute))
+	zlog.Info(ctx).Msgf("initial update in %s", d)
+	t := time.NewTimer(d)
 	defer t.Stop()
-
 	for {
 		select {
 		case <-ctx.Done():
 			return ctx.Err()
 		case <-t.C:
-			zlog.Info(ctx).Msg("starting update")
+			isInitial := u.known == nil
```

Comment:
Protect from what?

Reply:
We are making the check from within the goroutine that write to it.

Reply:
you're right. This seems safe to me. Can you add a comment above this just to mention it's safe because we know this is the only goroutine which writes to this slice?